content/fundamentals/advanced.md

```diff
@@ -0,0 +1,93 @@
+### Advanced Concepts
+
+So far we've breezed over a couple of advanced topics in TypeScript like decorators and how the DI system is actually working under the hood. If all you want is a system that works, ou can probably move on to another section, but if you want to get a deeper understanding of how Nest works, this is the section for you.
```

Comment:
Typo "ou" --> "you"

Reply:
@carnesen , to mark the places in the text where to make changes do this:
```suggestion
So far we've breezed over a couple of advanced topics in TypeScript like decorators and how the DI system is actually working under the hood. If all you want is a system that works, you can probably move on to another section, but if you want to get a deeper understanding of how Nest works, this is the section for you.
```